worker/lease/manifold.go

```diff
@@ -155,11 +144,13 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 }
 
 // NewWorker wraps NewManager to return worker.Worker for testability.
-func NewWorker(config lease.ManagerConfig) (worker.Worker, error) {
-	return lease.NewManager(config)
+func NewWorker(config ManagerConfig) (worker.Worker, error) {
+	return NewManager(config)
 }
 
 // NewStore returns a new lease store based on the input config.
-func NewStore(config lease.StoreConfig) *lease.Store {
-	return lease.NewStore(config)
+func NewStore(runner coredatabase.TxnRunner) lease.Store {
```

Comment:
I would do this:
```go
func NewStore(getter coredatabase.DBGetter) lease.Store {
    return service.NewService(domain.NewTxnRunnerFactoryForNamespace(getter.GetDB, coredatabase.ControllerNS))
}
```

Then further up you don't get the DB; it's just:
```go
store := s.config.NewStore(dbGetter)
```

Reply:
Yeah, that is better.